### ISSUE TITLE
compile problem on OSX

### ISSUE THREAD

**Issue by seven1240** [vlang/v#162]
nearly all examples fails except hello_world

```
$ ./v users_gui.v
V 0.0.1
Checking for updates...
V is up to date
errno=2 err='No such file or directory'
V panic: /Users/alex/code/lang//ui doesnt exist

$ ./v tetris.v
V 0.0.1
Checking for updates...
V is up to date
errno=2 err='No such file or directory'
V panic: /Users/alex/code/lang//rand doesnt exist
```

**Comment by medvednikov** [comment]
0.0.2 is out, please try it.

**Comment by seven1240** [comment]
```
dujinfang@seven7:~/workspace/v/macos$ ./v users_gui.v
V 0.0.2
Checking for updates...
V is up to date
errno=2 err='No such file or directory'
V panic: /var/tmp/lang//vlib/ui doesnt exist
dujinfang@seven7:~/workspace/v/macos$ ./v word_counter.v
V 0.0.2
Checking for updates...
V is up to date
errno=2 err='No such file or directory'
V panic: /var/tmp/lang//vlib/os doesnt exist
```

**Comment by seven1240** [comment]
I fixed the hello world one but others still complains missing.

```
dujinfang@seven7:~/workspace/v/macos$ cp -R vlib/builtin /var/tmp/lang/vlib/
dujinfang@seven7:~/workspace/v/macos$ ./v hello_world.v
V 0.0.2
Checking for updates...
V is up to date

real	0m1.731s
user	0m0.057s
sys	0m0.649s
dujinfang@seven7:~/workspace/v/macos$ ./v tetris.v
V 0.0.2
Checking for updates...
V is up to date
errno=2 err='No such file or directory'
V panic: /var/tmp/lang//vlib/rand doesnt exist
dujinfang@seven7:~/workspace/v/macos$ ls /var/tmp/lang/vlib/rand.
rand.o  rand.v
```

**Comment by medvednikov** [comment]
@seven1240 I forgot to force V to clean up `/var/tmp/lang`

Do this manually:

`rm -rf /var/tmp/lang`

I'll fix this in 0.0.3 today.

**Comment by seven1240** [comment]
@medvednikov thanks, it fixes the word_counter, but others still fails

```
dujinfang@seven7:~/workspace/v/macos$ ./v users_gui.v
V 0.0.2
Checking for updates...
V is up to date
errno=2 err='No such file or directory'
V panic: /var/tmp/lang//vlib/ui doesnt exist

dujinfang@seven7:~/workspace/v/macos$ ./v tetris.v
V 0.0.2
Checking for updates...
V is up to date

real	0m0.178s
user	0m0.029s
sys	0m0.043s
V panic:
dujinfang@seven7:~/workspace/v/macos$ ./v js.v
V 0.0.2
Checking for updates...
V is up to date

real	0m0.076s
user	0m0.028s
sys	0m0.041s
V panic:
```

**Comment by medvednikov** [comment]
Fixed in 0.0.3-0.0.5.

**Comment by seven1240** [comment]
Sorry I have to reopen this as 0.0.4 still panic on some default examples

```
dujinfang@seven7:~/workspace/v/macos$ ./v users_gui.v
V 0.0.4
Checking for updates...
V is up to date

real	0m0.057s
user	0m0.030s
sys	0m0.020s
V panic:
dujinfang@seven7:~/workspace/v/macos$ ./v js.v
V 0.0.4
Checking for updates...
V is up to date

real	0m0.053s
user	0m0.029s
sys	0m0.019s
V panic:
```
